verification/curator-service/ui/src/components/BulkCaseForm.tsx

```diff
@@ -79,14 +79,27 @@ interface ParsedCase {
     admin1?: string;
     admin2?: string;
     admin3?: string;
-    latitude?: string;
-    longitude?: string;
+    latitude?: number;
+    longitude?: number;
     locationName?: string;
 
     // Bulk upload specific data
     caseCount?: number;
 }
 
+type RecursivePartial<T> = {
+    [P in keyof T]?: RecursivePartial<T[P]>;
```

Comment:
🤔  This is pretty confusing, could you add a comment to the type describing it's use?

Reply:
Yeah, sorry about that. Done.